internal/tests_config.go

```diff
@@ -71,30 +40,9 @@ type TestsConfig struct {
 
 // Tests are the tests from yaml category pass / fail / skip.
```

Comment:
Let's add here rules: if  both `NameRegexPattern` and `OutputRegexPattern` match, parsing file fails.